home/templates/geneds.html

```diff
@@ -20,8 +20,14 @@
 		margin: 0 auto;
 	}
 
-	.dataTables_filter {
-		float: left !important;
+	@media (min-width: 768px) {
```

Comment:
note for reviewers: generally we use 800px as the mobile breakpoint, but 768 (bootstrap's md breakpoint) is better here to align with the md columns on the page.

We should probably move our breakpoint to 768 to avoid any confusion about this in the future.